dali/kernels/test/simd_test.cc

```diff
@@ -50,7 +50,7 @@ void TestConvertLoad(In lo, In hi) {
   int lanes = 16 / sizeof(In);
   In in[lanes];  // NOLINT
   for (int i = 0; i < lanes; i++) {
-    in[i] = lo + (hi - lo) * i / (lanes - 1);
+    in[i] = int64_t(lo) + (int64_t(hi) - int64_t(lo)) * i / (lanes - 1);
```

Comment:
This function can (at least theoretically) be used with floating point input. It's better to use `int64_t i` which will promote the multiplication to int64 for smaller types but maintain original floating point and signedness.
```suggestion
  for (int64_t i = 0; i < lanes; i++) {
    in[i] = lo + (hi - lo) * i / (lanes - 1);
```